pkgs/tools/misc/vdirsyncer/default.nix

```
@@ -1,27 +1,28 @@
-{ stdenv, python3Packages, fetchpatch, glibcLocales, rustPlatform, pkgconfig, openssl, Security }:
+{ stdenv, python3Packages, fetchFromGitHub, fetchpatch, glibcLocales, rustPlatform, pkgconfig, openssl, Security }:
 
 # Packaging documentation at:
 # https://github.com/untitaker/vdirsyncer/blob/master/docs/packaging.rst
-let
-  pythonPackages = python3Packages;
-  version = "0.17.0a3";
+python3Packages.buildPythonApplication rec {
+  version = "2018-08-05";
```

```suggestion
  version = "unstable-2018-08-05";
```

When version is not a release but a commit from a repository we use this format.

---

Thanks, forgot about that.